### PR TITLE
Create c++_code_fixes

### PR DESCRIPTION
**Pull request by Bhagyanekraje**
In this file I have written a c++ code to find the reverse of a string without using functions.

#### Checklist

- [ ] Added description of change
- [ ] Added file name matches [File name guidelines](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/CONTRIBUTING.md#New-File-Name-guidelines)
- [ ] Added tests and example, test must pass
- [ ] Added documentation so that the program is self-explanatory and educational - [Doxygen guidelines](https://www.doxygen.nl/manual/docblocks.html)
- [ ] Relevant documentation/comments is changed or added
- [ ] PR title follows semantic [commit guidelines](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/CONTRIBUTING.md#Commit-Guidelines)
- [ ] Search previous suggestions before making a new one, as yours may be a duplicate.
- [ ] I acknowledge that all my contributions will be made under the project's license.

Notes: 

<a href="https://gitpod.io/#https://github.com/TheAlgorithms/C-Plus-Plus/pull/1584"></a>

### CODE REVIEW

**Review by mishraabhinn — CHANGES_REQUESTED** [review]
[Refer](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/CONTRIBUTING.md)
@Bhagyanekraje

File: search/linear_search.cpp [hunk 1]
```diff
@@ -24,7 +24,7 @@ int LinearSearch(int *array, int size, int key) {
 }
 
 /** main function */
-int main() {
+void main() {
```
Comment (mishraabhinn) [inline search/linear_search.cpp:27]: Why?

**Review by mishraabhinn — CHANGES_REQUESTED** [review]

**Comment by mishraabhinn** [comment]
@Bhagyanekraje Please do check the relevant checkbox, and please do refer to the contribution guidelines before commiting. For any question you can ask here or  do message your doubts on [Gitter](https://gitter.im/TheAlgorithms) Thanks.  
Screenshot 2021-09-05 at 12 44 59

**Review by Bhagyanekraje — COMMENTED** [review]
> @Bhagyanekraje Please do check the relevant checkbox, and please do refer to the contribution guidelines before commiting. For any question you can ask here or do message your doubts on [Gitter](https://gitter.im/TheAlgorithms) Thanks.
> Screenshot 2021-09-05 at 12 44 59

ok sir

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Hey there! Thank you for submitting a PR! 😄 Sadly, this does not meet our requirements. Check out our [repository standards](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/CONTRIBUTING.md). Please read them **carefully** and **follow them**. Let us know if you need any help in our [**Discord server**](https://discord.gg/c7MnfGFGa6).

**Pull request closed without merging**